data/World/Forest Temple.json

```diff
@@ -36,6 +55,7 @@
         "exits": {
             "Forest Temple NE Outdoors": "(Progressive_Scale, 2)",
             #Other methods of crossing through the well are not currently relevant.
+            "Forest Temple Well" : "can_use(Iron_Boots) or at('Forest Temple Outdoors High Balconies', True)",
```

Comment:
If you can use the Iron Boots in either courtyard, you will have access to the upper balconies. You can Iron Boots through the well if necessary, then climb the vines, and then adult can kill the bubble and hit the well water switch. So the Iron Boots check here is not actually doing anything, with the way forest temple is currently structured. So you could just put the recovery hearts these well exits lead to in the upper balconies region, same as the well chest is now.

I admit having the contents of the well be in the upper balconies region is strange.